tools/package_tools/mmdeploy_builder.py

```diff
@@ -292,33 +316,52 @@ def create_package(cfg: Dict, mmdeploy_dir: str):
             _copy(install_dir, sdk_tar_dir)
             _remove_if_exist(osp.join(sdk_tar_dir, 'example', 'build'))
 
-            # create sdk python api wheel
-            # for linux
-            python_api_lib_path = glob(
-                osp.join(mmdeploy_dir, 'build/lib/mmdeploy_python.*.so'))
-            # for windows
-            python_api_lib_path += glob(
-                osp.join(mmdeploy_dir, 'build/bin/*/mmdeploy_python.*.pyd'))
-            num_libs = len(python_api_lib_path)
-            if num_libs != 1:
-                logging.info('find multiple mmdeploy_python libraries.')
-            python_api_lib_path = python_api_lib_path[0]
-
-            sdk_python_package_dir = osp.join(build_dir, '.mmdeploy_python')
-            _copy(PACKAGING_DIR, sdk_python_package_dir)
-            _copy(
-                osp.join(mmdeploy_dir, 'mmdeploy', 'version.py'),
-                osp.join(sdk_python_package_dir, 'mmdeploy_python',
-                         'version.py'))
-            _copy(python_api_lib_path,
-                  osp.join(sdk_python_package_dir, 'mmdeploy_python'))
-            sdk_wheel_dir = osp.abspath(osp.join(sdk_tar_dir, 'python'))
-            bdist_cmd = _create_bdist_cmd(
-                cfg, c_ext=True, dist_dir=sdk_wheel_dir)
-            _call_command(bdist_cmd, sdk_python_package_dir)
-
-            # remove temp package dir
-            _remove_if_exist(sdk_python_package_dir)
+            # build SDK Python API according to different python version
+            # 1. get conda env path from python's path.
+            # We assume python is managed by conda
+            python_versions = ['3.6', '3.7', '3.8', '3.9']
+            for python_version in python_versions:
+                python_executable = shutil.which(f'python{python_version}')
+                if python_executable:
+                    logging.info(python_executable)
+                    conda_env_path = osp.dirname(
+                        osp.dirname(osp.dirname(python_executable)))
+                    logging.info(conda_env_path)
+                    break
+
+            for python_version in python_versions:
+                python_major, _, python_minor = list(python_version)
```

Comment:
Use `packaging.version.parse` instead of char list. So we can add 3.10+ in future.